packages/next-css/readme.md

```diff
@@ -195,6 +195,35 @@ Create a CSS file `style.css` the CSS here is using the css-variables postcss pl
 
 When `postcss.config.js` is not found `postcss-loader` will not be added and will not cause overhead.
 
+### Leverage long-term browser caching of style.css in production
+
+For production builds, the hashed filename is available in `this.props.buildManifest.css`, so the `_document.js` could be changed to:
+
+```js
+// ./pages/_document.js
+import Document, { Head, Main, NextScript } from 'next/document'
+
+export default class MyDocument extends Document {
+  render() {
+    const { buildManifest } = this.props;
+    return (
+      <html>
+        <Head>
+          {buildManifest.css.map(assetPath => (
+            // this will work in both development and production
```

Comment:
Sounds like this comment is contradicting the earlier mention you made:

> For production builds, the hashed filename is available in `this.props.buildManifest.css`, so the `_document.js` could be changed to:

To be clear buildManifest works in both development and production mode as far as I know 👍

Reply:
Agreed, it is a bit confusing. I rewrote it 079a4fa. Let me know if it's still confusing. :)